frontend/src/components/status/index.vue

```diff
@@ -52,13 +77,13 @@ const loadingStatus = [
     'removing',
     'applying',
     'uninstalling',
+    'downloading',
+    'packing',
+    'sending',
+    'waiting',
 ];
 
 const loadingIcon = (status: string): boolean => {
     return loadingStatus.indexOf(status) > -1;
 };
-
-onMounted(() => {
-    status.value = props.status.toLocaleLowerCase();
-});
 </script>
```

Comment:
There are no immediate issues with this code, but it is worth noting that Vue's `ref` usage should ideally be avoided due to performance concerns.

Here are some observations:

1. It seems that the use of `defineProps()` has been removed as it was deprecated since Vue 3.x and replaced with `<props/>`. This change may have implications elsewhere across your project.
  
2. The component includes a custom prop "msg". However, there doesn't seem to be any corresponding logic defined within it. If you decide to proceed using custom props, remember to update all occurrences in the relevant places like event handlers.

For optimizing further, make sure to leverage V-Model feature when dealing with multiple text fields or components where data binds between UI elements directly rather than through props. 

The tag-based approach appears fine here, though consider moving the loading status condition into a state management module for reusability.

Remember always testing thoroughly before committing substantial changes.